### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by robgg2011**
Adds cypress testing to application

### CODE REVIEW

File: cypress.config.ts
Comment (annaholdenjacobsen) [inline cypress.config.ts:1]: Mabye move the cypress folder into a new folder called test! Mabye we need to write snapshot tests as well so it is nice to have all the testing in one place

File: cypress/e2e/HomePage.cy.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('Home page', () => {
+  it('successfully loads', () => {
+    cy.visit('/')
+  })
+})
```
Comment (petr7555) [inline cypress/e2e/HomePage.cy.ts]: Missing newline at the end of the file.

File: cypress/component/BackendHealthCheck.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+describe('BackendHealthCheck.cy.tsx', () => {
```
Comment (petr7555) [inline cypress/component/BackendHealthCheck.cy.tsx]: Do we need test that does not test anything?

File: cypress.config.ts
Comment (petr7555) [reply cypress.config.ts:1]: I'd say no, stick with the structure cypress generated when setting it up.

File: README.md [hunk 3]
```diff
@@ -11,3 +11,13 @@ The deployment is automatic on push to the `master` branch.
 - `npm start`
 
 The frontend will be running on http://localhost:3000/.
+
+## Run tests
```
Comment (petr7555) [inline README.md]: Can you create a testing section containing instructions on running **Unit tests**, **E2E tests** and **Component tests**?

File: .github/workflows/lint-and-test.yml [hunk 4]
```diff
@@ -29,3 +29,17 @@ jobs:
 
       - name: Test
         run: npm run test
+
+  end-to-end-test:
```
Comment (petr7555) [inline .github/workflows/lint-and-test.yml:33]: Can we have 2 jobs running in parallel, one for **E2E tests** and the other for **Component tests**?

**Comment by petr7555** [comment]
Would you please create at least one E2E test and one Component test that does something meaningful (some action, assertion) so we can see if it is set up correctly?

File: cypress/component/BackendHealthCheck.cy.tsx (hunk 2 above)
Comment (robgg2011) [reply cypress/component/BackendHealthCheck.cy.tsx]: Just a placeholder / sample

**Pull request closed without merging**